idunn/utils/categories.yml

```
@@ -93,22 +106,21 @@ categories:
     service:
         pj_what: "poste"
         pj_filters:
-            - "plombiers"
-            - "garages automobiles"
             - "envoi, distribution de courrier, de colis"
-            - "serrurerie, métallerie"
-            - "dépannage serrurerie"
-            - "mairies"
-            - "services de gendarmerie, de police"
-            - "sapeurs-pompiers, centres de secours"
-            - "préfectures, sous-préfectures"
         raw_filters:
             - "car_repair,*"
             - "post_office,*"
             - "townhall,*"
             - "police,*"
             - "fire_station,*"
-        regex: "service|car repair|post office|police|garage|poste|gendarmerie|pompiers"
+        regex: "(^| )post"
+    police:
+        pj_what: "police"
+        pj_filters:
+            - "services de gendarmerie, de police"
+        raw_filters:
+            - "police,*"
+        regex: "(^| )(poli|gendar)"
```

This regex may trigger false positives for queries such as 
```
polish embassy
institut d'études politiques
policlinico
```
```suggestion
        regex: "(^| )(police|gendar)"
```